llm/gguf.go

```diff
@@ -552,7 +554,10 @@ func WriteGGUF(ws io.WriteSeeker, kv KV, ts []Tensor) error {
 	}
 
 	var alignment int64 = 32
-	for _, t := range ts {
+	for i, t := range ts {
+		fn(api.ProgressResponse{
+			Status: fmt.Sprintf("converting model %d%%", 100*(i+1)/len(ts)),
```

Comment:
unlikely but we should guard against `len(ts)==0`

Reply:
I think it should be impossible for len(ts) to be 0 right? Since if it was 0, the `for loop` wouldn't run